spec/filters/spec_helper.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
```

Comment:
I don't think you need this - I can run the tests locally without starting the derby server in this way

Reply:
We don't for now, but will soonish. I want to keep it though.